test/src/test/java/org/corfudb/infrastructure/LogUnitCacheTest.java

```diff
@@ -39,6 +39,13 @@ public AbstractServer getDefaultServer() {
                 .build());
     }
 
+    @Override
+    public void setServer(AbstractServer server) {
+        router.reset();
+        router.setIgnoreMessageValidation(true);
```

Comment:
Why ignore the epoch/cluster id check?

Reply:
Because for some tests we just test a specific server, and not the stack. It's easier to ignore the validation in the router then to bootstrap a node with a specific layout, since epoch or a presence of a layout or a cluster id is irrelevant in the test.

Reply:
Edit: Was removed because was inconsistent in a couple of places. Now we explicitly bootstrap in the test cases, set cluster Ids and epochs on client and server routers.